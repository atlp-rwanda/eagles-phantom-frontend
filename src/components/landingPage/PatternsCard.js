import React,{ Component } from 'react';
import '../../scss/homeContent.scss';

const Pattern=({description})=>{
    return(
      <div className="patterns">
      <p>{description}</p>
      </div>
    )
  }

  export default Pattern;