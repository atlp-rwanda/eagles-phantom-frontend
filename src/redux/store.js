import {createStore, applyMiddleware } from 'redux';
import { composeWithDevTools} from 'redux-devtools-extension';
import thunk from 'redux-thunk';
import promise from 'redux-promise-middleware';
import allReducers from './reducers/';

// const rootReducer = combineReducers({
//   count: counterReducer,
// });

// const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const middleware = applyMiddleware(thunk, promise)
const store = createStore(allReducers, {}, composeWithDevTools(middleware));

export default store;
