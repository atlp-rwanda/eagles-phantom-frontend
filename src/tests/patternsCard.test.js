import React from 'react';
import { Provider } from "react-redux";
import store from "../redux/store.js";
import { render } from '@testing-library/react';
import PatternsCard from '../components/landingPage/PatternsCard';

const patternsCard=()=>{
    return render(
       <Provider store={store}>
       <PatternsCard />
       </Provider>
    )
}
describe("PatternsCard",()=>{
    it("shoould render the component",()=>{
        const { asFragment }=patternsCard();
        expect(asFragment(<PatternsCard />)).toMatchSnapshot()
    });
})