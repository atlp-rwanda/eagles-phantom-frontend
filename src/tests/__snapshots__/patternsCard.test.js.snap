// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PatternsCard shoould render the component 1`] = `
<DocumentFragment>
  <div
    class="patterns"
  >
    <p />
  </div>
</DocumentFragment>
`;
